packages/demo-cards/claim.gts

```diff
@@ -30,11 +32,11 @@ class Isolated extends Component<typeof Claim> {
         /></FieldContainer>
       <FieldContainer @label='Chain'><@fields.chain /></FieldContainer>
       {{#if this.connected}}
-        <Button>
+        <Button {{on 'click' this.claim}}>
```

Comment:
this looks good. also for bonus points, ember concurrecny makes loading states _really_ easy. you can have:
```
{{#if this.initialize.isRunning}}
  Initializing...
{{/if}}
if you want to show something meaningful while the initialization is happening.

Reply:
resolved https://github.com/cardstack/boxel/pull/353/commits/5bfdf2e6c2a73773ac263ed6a1cb39b20785afdd